### PR TITLE
Initial pull request CMDB Collector 

### PR DESCRIPTION
**Pull request by Sbrenthughes**
Initial pull request for CMDB Collector

### CODE REVIEW

File: core/src/main/java/com/capitalone/dashboard/util/FeatureCollectorConstants.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
         public static final String JIRA = "Jira";
         public static final String VERSIONONE = "VersionOne";
         public static final String GITLAB = "GitlabFeature";
+        public static final String HPSM = "Hpsm";
```
Comment (satishc1) [inline core/src/main/java/com/capitalone/dashboard/util/FeatureCollectorConstants.java]: @Sbrenthughes This one is probably not needed.

Comment (Sbrenthughes) [inline core/src/main/java/com/capitalone/dashboard/util/FeatureCollectorConstants.java]: Updated. Thank you

File: collectors/cmdb/hpsm/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+COPY *.jar /hygieia/
+COPY github-properties-builder.sh /hygieia/
+
+WORKDIR /hygieia
+
+VOLUME ["/hygieia/logs"]
+
+CMD ./github-properties-builder.sh && \
+  java -jar github-scm-collector*.jar --spring.config.location=/hygieia/hygieia-github-scm-collector.properties
+
```
Comment (satishc1) [inline collectors/cmdb/hpsm/docker/Dockerfile]: @Sbrenthughes Needs to be updated for hpsm

File: collectors/cmdb/hpsm/docker/github-properties-builder.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```
Comment (satishc1) [inline collectors/cmdb/hpsm/docker/github-properties-builder.sh]: @Sbrenthughes Needs to be updated for hpsm

File: collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/HpsmClient.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.model.Cmdb;
+
+import java.util.List;
+
+/**
+ * Client for fetching commit history from GitHub
```
Comment (satishc1) [inline collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/HpsmClient.java]: @Sbrenthughes update

File: collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/HpsmCollectorTask.java [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * CollectorTask that fetches Commit information from GitHub
+ */
```
Comment (satishc1) [inline collectors/cmdb/hpsm/src/main/java/com/capitalone/dashboard/collector/HpsmCollectorTask.java]: @Sbrenthughes update

File: collectors/cmdb/hpsm/src/main/resources/logback.xml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+ <appender name="ROLLING" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+      <!-- rollover daily -->
+      <fileNamePattern>logs/github-collector-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+      <timeBasedFileNamingAndTriggeringPolicy
```
Comment (satishc1) [inline collectors/cmdb/hpsm/src/main/resources/logback.xml]: @Sbrenthughes change log file name

**Comment by satishc1** [comment]
@tabladrum LGTM

**Pull request merged**